.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Support request
+about: Need clarification on a few scripts or the pipeline in general? Ask us!
+---
+
+<!--
+Hi!  Read this; it's important.
+
+This is an issue tracker for Autometa.  File support requests about
+Autometa here.
+-->
+
+### User checklist
+
+- [ ] Are you using the latest release?
+- [ ] Are you using python 3?
+- [ ] Did you check previous issues to see if this has already been mentioned?
+- [ ] Are you using a Mac or Linux machine?
+
+#### Description
+
+<!-- A clear and concise description of what the support request is about. -->
+
+#### Expected Behavior
+
+<!-- Please describe the behavior you are expecting. -->
+
+#### System Environment
+
+<!-- Please provide relevant system environment information. -->
+
+- Operating System:
+- RAM:
+- Disk:
+
+#### Files
+
+<!-- A list of relevant files for this request. This will help people navigate the project and offer some clues of where to start. -->
+
+#### Tasks/Command(s)
+
+<!-- If this issue is describing steps for your workflow, include specific tasks in the order they need to be done. Include links to specific lines of code where you need clarification for each respective task. -->
```

Comment:
When you say "include links to specific lines of code" - do you mean in error messages? I feel that this current wording is a little unfriendly to people who might not know what you mean. Perhaps you could instead ask for the contents of the log file including error messages?

Reply:
I was thinking lines in the code base. This template seems like a mix between a bug report and a feature request. I'll add a section on reporting any error messages and change the wording on the lines of code to more of a suggestion/optional if you know where the code lives sort of comment